lib/grpcx/server.rb

```diff
@@ -22,6 +21,29 @@ def handle(service)
 
         super(service)
       end
+
+      private
+
+      def apply_default_options(opts={})
+        opts.tap do
+          opts[:pool_size] ||= ENV['GRPC_SERVER_THREADS'].to_i if ENV['GRPC_SERVER_THREADS']
+          opts[:max_waiting_requests] ||= ENV['GRPC_SERVER_QUEUE'].to_i if ENV['GRPC_SERVER_QUEUE']
+
+          # interceptors are fired in FIFO order,
+          # so more generic handlers come last:
+          (opts[:interceptors] ||= []).concat(default_interceptors)
+        end
+      end
+
+      def default_interceptors
```

Comment:
1. Same, this can live outside of methods
2. Slightly unorthodox, but you should require interceptors here. Otherwise, if grpcx is loaded after e.g. activerecord, it will not be defined at load time

Reply:
Well, we don't need `ActiveRecord` to be present when interceptor files are evaluated. `ActiveRecord::*` is used only in methods (so used only when server starts - usually, AR should be required at that point).

So you mean something like this, right?

```
def default_interceptors
  require 'grpcx/server/interceptor'
  [ ... ]
end
```

I can do it if you insist, but I don't think this is really needed, top-level require should be fine.